### DISCUSSION TITLE
deno support 🥺

### DISCUSSION THREAD

**Discussion by foxydevloper** [pulumi/pulumi#12559] [Ideas]
This project is awesome, as someone coming from terraform. Could we possibly get deno support? I'm sure it could take advantage of deno's new built in npm support, using deno instead of node would get rid of a lot of the boilerplate when creating a new pulumi project (node_modules, package.json, package-lock.json, tsconfig.json)

**Comment by ericrudder** [comment]
Thanks for the feedback; this is a dupe of https://github.com/pulumi/pulumi/issues/3838 if you'd like to work on it!
